app/3.0/docs/devguide/style-shadow-dom.md

```diff
@@ -429,260 +471,273 @@ And you can select by slot name:
     <p>
       <slot name='para2'></slot>
     </p>
-  </template>
-  ...
-</dom-module>
+  `;
+}
+...
 ```
 
-`index.html` { .caption}
-```html
-<link rel="import" href="x-foo.html">
+index.html { .caption }
 
-<x-foo>
-  <div slot="para1">I'm green.</div>
-  <div slot="para2">I'm not green.</div>
-</x-foo>
+```html
+<head>
+  <script type="module" src="custom-element.js"></script>
+</head>
+<body>  
+  <custom-element>
+    <div slot="para1">I'm green.</div>
+    <div slot="para2">I'm not green.</div>
+  </custom-element>
+</body>
 ```
 
+[See it on Plunker](http://plnkr.co/edit/hR3I4w?p=preview)
+
 ## Share styles between elements
 
 ### Use style modules {#style-modules}
 
 The preferred way to share styles is with *style modules*. You can package up styles in a style module, and share them between elements.
 
```

Comment:
Going forward, we're probably going to recommend using template mechanism rather than style modules. We should add that as a separate update after this one, though, I think.

Reply:
ok, created #2513 to track